### PR TITLE
Basicstructure

### PR DESCRIPTION
**Pull request by Grifo89**
This is our feature branch and we are requested a feedback of how is it. Thanks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,176 @@
...
 <html lang="en" dir="ltr">
   <head>
-    <meta charset="utf-8">
-    <link rel="stylesheet" href="/style.css">
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="./reset.css" />
+    <link rel="stylesheet" href="./style.css" />
+    <!-- <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
```
Comment (wathigo) [inline index.html]: **[REQUIRED]** Remove the commented code

**Review by wathigo — CHANGES_REQUESTED** [review]
## Hi @Grifo89  and @OlukaDenis 
### You have done great work for the project! :clap:,
Just some minor changes required to mark this as complete. 
1. Your code contains some errors. Upload your files to [w3 validator](https://validator.w3.org/#validate_by_input+with_options) and fix all the errors.
2. Remove the commented code below :point_down:

**Please do the required changes then submit another request for review!**

File: index.html (hunk 1 above)
Comment (Grifo89) [reply index.html]: Thanks Wathigo, We're gonna make those changes you suggest!

Comment (Grifo89) [reply index.html]: Changes done!

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Pull request under review by: Inyama, Uchechukwu Henry.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Hi team :wave: good job. 
#### You have implemented the changes as suggested by @wathigo, but there changes my colleague missed to mention :hugs: 
- [ ] There is no .stickler.yml file in this project. It should be added at the root directory of this project. Visit this [link](https://github.com/microverseinc/linters-config/tree/master/css) for instructions on how to set it up.

- [ ] Also, make your README.md file more descriptive. As pointers :dart: Include the following to your README.md :
    *     A short introduction of what the project is about.
    *      You can add the author's of the project.
    *      You can add the link of clone 
    *      You can also add the link to Odin project for a detailed description of the project.
    *       You can include the links of github profile.

[**Required**] When creating your next pull request or after effecting a change as requested from a TSE, add comment, the described the changes you implemented :smiley: 


### After implementing this changes, submit another form for code review.

### This project is not approved :stop_sign:

**Comment by OlukaDenis** [comment]
We have already made the changes that you suggested. Please, we request for a code review. Thanks

File: index.html [hunk 2]
```diff
@@ -1,11 +1,171 @@
...
+          <i class="fas fa-bars"></i>
+          <img
+            src="https://upload.wikimedia.org/wikipedia/commons/e/e1/Logo_of_YouTube_%282015-2017%29.svg"
+            alt=""
+            class="logo-image"
+          />
+        </div>
+        <div class="search">
+          <input type="text" placeholder="Search" />
+          <i class="fas fa-search"></i>
```
Comment (Ispirett) [inline index.html:22]: Please change this to a button `<button>` Tag

**Review by Ispirett — APPROVED** [review]
You did excellent work with this project Folks and the changes made  :+1:  :1st_place_medal: 

image

Before merge please make this change and then merge and complete this project.

**Pull request merged**